docs/index.js

```diff
@@ -435,6 +442,19 @@ async function initPayment() {
               "background-color: #FF4800; display: flex;",
               1500
             );
+          // Affirm payment component error - we want to unmount the component and hide payment method
+          case "affirm":
+            try {
+              checkout.remove("affirm");
+              showPaymentMethod(false, "affirm");
+            } catch (e) {
+              console.log(e);
+            }
+            showMessage(
+              `onBeforeError called in Affirm`,
+              "background-color: #FF4800; display: flex;",
+              1500
+            );
```

Comment:
niggle: this is kinda replicated code... may be we can extract it into special one?